akka-samples/akka-sample-distributed-data-java/src/main/java/sample/distributeddata/ReplicatedMetrics.java

```diff
@@ -0,0 +1,167 @@
+package sample.distributeddata;
```

Comment:
the reason is that we don't want to scare people by typesafe copyrights in samples